src/shell/shell_cmds.cpp

```diff
@@ -2242,3 +2243,259 @@ void DOS_Shell::CMD_VOL(char* args)
 	const auto trimmed_label = trim(label);
 	WriteOut(MSG_Get("SHELL_CMD_VOL_OUTPUT"), drive, trimmed_label, high, low);
 }
+
+void DOS_Shell::CMD_MOVE(char* args)
+{
+	HELP("MOVE");
+
+	// TODO: Add support for these flags along with the COPYCMD environment
+	// variable (overwrite prompts) Also currently ignored by CMD_COPY
+	(void)ScanCMDBool(args, "Y");
+	(void)ScanCMDBool(args, "-Y");
+
+	char* rem = ScanCMDRemain(args);
+	if (rem) {
+		WriteOut(MSG_Get("SHELL_ILLEGAL_SWITCH"), rem);
+		return;
+	}
+
+	bool parsing          = true;
+	bool quote            = false;
+	bool stop_next_letter = false;
+	bool comma            = false;
```

Comment:
`curr_token_is_comma` or something, but with the proposed better parsing logic (see below) this would disappear.

Reply:
I renamed it to `continue_after_stop` since that's really what it's doing.  It's a flag for whether the parser should continue parsing the next word or if it should finish.  If it sees a comma in between the words, it continues.  If not, it stops.